src/app/ubs/ubs-admin/ubs-admin.component.ts

```diff
@@ -11,13 +16,32 @@ import { takeUntil } from 'rxjs/operators';
 })
 export class UbsAdminComponent implements OnInit, OnDestroy {
   private destroy: Subject<boolean> = new Subject<boolean>();
+  hasAuthorities: boolean;
+  authorities: string[] = [];
+  permissions$ = this.store.select((state: IAppState): Array<string> => state.employees.employeesPermissions);
 
-  constructor(private translate: TranslateService, private localStorageService: LocalStorageService) {}
+  constructor(
+    private translate: TranslateService,
+    private localStorageService: LocalStorageService,
+    private store: Store<IAppState>,
+    public ubsAdminEmployeeService: UbsAdminEmployeeService,
+    public jwtService: JwtService
+  ) {}
 
   ngOnInit() {
     this.localStorageService.languageBehaviourSubject.pipe(takeUntil(this.destroy)).subscribe((lang) => {
       this.translate.setDefaultLang(lang !== 'en' && lang !== 'ua' ? 'ua' : lang);
     });
+    const userEmail = this.jwtService.getEmailFromAccessToken();
+    this.store.dispatch(GetEmployeesPermissions({ email: userEmail }));
+    this.authoritiesSubscription();
+  }
+
+  private authoritiesSubscription() {
+    this.permissions$.subscribe((authorities) => {
+      this.hasAuthorities = authorities.length > 0;
+      this.authorities = authorities;
+    });
```

Comment:
add .pipe(takeUntill(this.destroy)).subscribe(...) to unsubscribe from Observable after destruction and return type void.